This file is part of mingw-cross-env.
See doc/index.html for further information.

Contains ad hoc patches for cross building.

From 87a791a05687c7737d1c50381a0de30ba1acf96e Mon Sep 17 00:00:00 2001
From: mingw-cross-env
Date: Mon, 24 Oct 2011 09:40:10 +0200
Subject: [PATCH] add pkg-config support

based on
http://old.nabble.com/-PATCH--Add-pkgconfig-file-td23312302.html
---
 Makefile.am  |    3 +++
 configure.ac |    1 +
 glut.pc.in   |   12 ++++++++++++
 3 files changed, 16 insertions(+), 0 deletions(-)
 create mode 100644 glut.pc.in

diff --git a/Makefile.am b/Makefile.am
index 39c5628..b504bc6 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -68,3 +68,6 @@ dist-hook:
 
 libtool: $(LIBTOOL_DEPS)
 	$(SHELL) ./config.status --recheck
+
+pkgconfigdir = $(libdir)/pkgconfig
+pkgconfig_DATA = glut.pc
diff --git a/configure.ac b/configure.ac
index 15dc997..2f25bcb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -119,5 +119,6 @@ AC_CONFIG_FILES([
 	progs/demos/spaceball/Makefile
 	progs/demos/subwin/Makefile
 	src/Makefile
+	glut.pc
 ])
 AC_OUTPUT
diff --git a/glut.pc.in b/glut.pc.in
new file mode 100644
index 0000000..ea57c4a
--- /dev/null
+++ b/glut.pc.in
@@ -0,0 +1,12 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@includedir@/@PACKAGE@
+
+Name: glut
+Description: A freely licensed alternative to the GLUT library
+Version: @VERSION@
+Libs: -L${libdir} -lglut
+Libs.private: -lglu32 -lopengl32 -lwinmm -lgdi32 -mwindows
+Cflags: -DFREEGLUT_STATIC -I${includedir}
+
-- 
1.7.8

